src/components/NewApiDelegationPage/NewApiDelegationPage.tsx

```diff
@@ -66,8 +95,20 @@ export const NewApiDelegationsPage = () => {
             <div className={classes.pageContent}>
               <h2>Gi tilgang til API</h2>
               <h3>Velg hvilke API du vil gi tilgang til ved å klikke på pluss-tegnet.</h3>
-              <div className={classes.searchField}>
-                <SearchField></SearchField>
+              <div className={classes.searchSection}>
+                <SearchField
+                  value={searchString}
+                  onChange={(e: any) => handleSearch(e.target.value)}
```

Comment:
Any should not be used unless there's a temporary reason for using it
```suggestion
                  onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
                    handleSearch(e.currentTarget.value)
                  }
```

Reply:
Agreed 😊